### ISSUE TITLE
Import whole set of translations in HOC

### ISSUE THREAD

**Issue by Boimb** [synacor/preact-i18n#27]
Hi,
Thx for that great tool !
I'm wondering if this is possible to import a whole set of "key": "value" with the HOC withText in functionnal component ? 
Today, i'm using it this way : 
```
const MyForm = withText({
  nameLabel: "form.labels.name",
  namePlaceholder: "form.placeholders.name"})( (props) => (<Form>
    <Label>{nameLabel}</Label>
    <Input>{namePlaceholder}</Input>
  </Form>))
```
I'd like to do "it this way" : 
```
const MyForm = withText({form: "form"})( (props) => (<Form>
    <Label>{form.labels.name}</Label>
    <Input>{form.placeholders.name}</Input>
  </Form>))
```
Is there a way of doing this ?
Thanks for your help.

**Comment by billneff79** [comment]
You can't currently do what you describe above because trying to have a key for a value in preact-i18n that is an object but not a pluralized field will try to apply the templating function to it and will throw an exception.  We could fix that, but there isn't really a use case for it, which I'll explain:

The more appropriate way to do what you describe above is with the `<Text />` component, e.g.:
```
const MyForm = ( (props) => (<Form>
    <Label><Text id="form.labels.name" /></Label>
    <Input><Text id="form.placeholders.name /"</Input>
  </Form>))
```

We typically only use `withText` in scenarios where the `<Text/>` component can't be used (i.e. where JSX won't be interpreted), which are typically text based attributes of native dom elements, like the `placeholder` attribute of a native `<input type="text" />` component.  You can't do `<input type="text" placeholder={<Text id="foo.bar"/>} />` and have the `Text` component interpreted (although wrapping that in a `<Localizer/>` component will actually make that work - see the README).  But if you use `withText` you can get your localized placeholder text as a prop and just pass that into the input component.

So, to recap, use `<Text />` wherever you can and use `withText` primarily for scenarios where JSX won't be interpreted
